### ISSUE TITLE
Add automation to connect repos

### ISSUE THREAD

**Issue by Sgtpluck** [18F/delivery-assurance#9]
**Project Name**
Delivery Assurance, Guides & Methods, Tock, Blog Posts.

**A brief description of the work**
In order to get the Delivery Assurance shared board working automatically, we need to add a GitHub action to  each of the repos we want to connect.

Example Action here:
https://github.com/18F/ux-guide/blob/main/.github/workflows/on_new_issue.yml

**Billable?**
- [x] Yes
- [ ] No
If yes, tock code:
1813

**Point of contact(s)**
Davida

**Acceptance Criteria**
When an issue is opened in each connected repo (todo: construct a list) it is automatically added to the Delivery Assurance shared Projects board.

**Timeline**
- [ ] Urgent deadline: Ex. EOD tomorrow (10/10/23)
- [x] Time-frame: Wed-Friday of next week

**Indicate which, if any, priority this ask falls under**
- [ ] Know, share, use and evolve our best practices
- [x] Increase 18F’s operational resilience
- [ ] Deliver more value to the public through our projects

**Skills needed**
- [x] Engineering

**Additional context**
The Repo list (Please add tickets and PRs as they are created)
```[tasklist]
- [ ] #16
- [ ] 18F/tock#1541
- [ ] 18F/blog-drafts#850
- [ ] 18F/eng-hiring#149
- [ ] 18F/18f.gsa.gov#3651
```

Guides & Methods -- currently on hold as we think about how this will work with the team being staffed
- [ ] [Accessibility Guide](https://github.com/18F/accessibility)
- [ ] [Agile Guide](https://github.com/18F/agile)
- [ ] [Content Guide](https://github.com/18F/content-guide)
- [ ] [Derisking Guide](https://github.com/18F/derisking-gov-tech)
- [ ] [Design Methods](https://github.com/18F/methods)
- [ ] [Engineering Practices](https://github.com/18F/development-guide)
- [ ] [Product Guide](https://github.com/18F/product-guide)
- [ ] [User Experience Guide](https://github.com/18F/ux-guide)
- [ ] [Visual Identity Guide](https://github.com/18F/brand)

**Comment by Sgtpluck** [comment]
state of the automation as of 2/16/2023:


The blog-drafts repo is blocked on figuring out who the admin of the repo is.
The eng-hiring repo is just waiting for the admins to be ready for it.

We've started a list of repos and other projects to consider as the team moves forward [here](https://docs.google.com/document/d/1VTwlBRdyc_FCnzjMocJnbxQAZj6m9qBrnAjrAkLAEF0/edit#)

**Comment by Sgtpluck** [comment]
eng-hiring admins have the PR and the secrets -- they will merge when they are ready!

**Comment by mgwalker** [comment]
I'd never thought of that! Great idea!

**Comment by mgwalker** [comment]
Of the "definitely do these" repos, [eng-hiring](/18F/eng-hiring) is the only one still outstanding. There's been no work with any of the guides and methods repos.

**Comment by colinmurphy01** [comment]
👏
